### ISSUE TITLE
To Do List

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [JessicaLucindaCheng/website#29]
[ ] Review [tamara-snyder's Assess safety of website from liquid injections #2141](https://github.com/hackforla/website/issues/2141)
[ ] [Review the alt text for Wins page images and ensure that they adhere to WCAG #2569](https://github.com/hackforla/website/issues/2569)
[ ] Write an issue to update https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#v-after-pull-request-is-submittedmerged
[ ] Write an issue to identify which VS Code plugin to use for future development -> spell check
[ ] For Getting Started page, audit h tags to use title classes and remove existing font properties in SCSS classes for the headers
[ ] Write issue to fix meeting times to be by day and time order on project profiles
[ ] Write dev epic for [Review the alt text for Credits page images and ensure that they adhere to WCAG #2572](https://github.com/hackforla/website/issues/2572)
[ ] Write an issue to edit line 41 your to you're Mobile_responsiveness.yml
[ ] Write an issue to edit line 33 `each changes` to `each change` in Mobile_responsiveness.yml
[ ] Write an issue to remove line 46 capitalize `figma` in Mobile_responsiveness.yml
[ ] Write an issue to remove line 5 comment `# assignees` from Mobile_responsiveness.yml
[ ] Draft: Audit and go through all project cards/info pages and remove  Docker and Dockerfile in technologies that are also listed as languages
[ ] Draft: Separate lang and tech filter tag on project page
[ ] Write issue related to [Review the alt text for Citizen Engagement page images and ensure that they adhere to WCAG #2570](https://github.com/hackforla/website/issues/2570)
[ ] Write issue to check all the links for the top nav bar if Adolf is finished with this issue https://github.com/hackforla/website/issues/2455 when the following issues are completed (see anchor issues listed in https://github.com/hackforla/website/issues/2393)
[ ] Write issue to refactor icon files into `iconography` folder
    * [ ] Change `site.data.internal.credits` to `site.data.internal.credits.iconography` for https://github.com/hackforla/website/blob/eb4c4c37a05fd6aad51c3d2b69cede73d07c584d/pages/credits.html#L25
[ ] Write issue for [displaying languages from a project that has multiple repos](https://github.com/hackforla/website/issues/2728#issuecomment-1086902818)
[ ] Projects page all menus in mobile have pointer be if you click it again it should close menu
[ ] Project page dropdown menus should be as wide as the mobile screen
[ ] Make an issue for [Also, for _wins-data.json, UniqueID shouldn't be there yet because the Google Apps Script related to issue #2385 hasn't been added to the product script.](https://github.com/hackforla/website/issues/2901#issuecomment-1104537430)
[ ] Check `label:"ready for dev leads"` for WCAG alt text reviews that devs need to write
[ ] Write feasibility issue for [Carlos's issue in regards to Galaxy Fold](https://github.com/hackforla/website/pull/2977#issuecomment-1094341044)
[ ] 🚩 Create issues related to [Develop Homepage Redesign #2317](https://github.com/hackforla/website/issues/2317#issuecomment-1083703541)
  [ ] Delete line 13 and uncomment line 14 when about page is done. Need to add a dependency label and figure out what the issue before this one will be. https://github.com/hackforla/website/blob/467a797f7c9518fcb466b6bdd034de27050b838e/_includes/home-page/hero.html#L12-L13
  [ ] Communities of Practice section images should have `alt=""` and remove `/` from the end of the img tag.
  [ ] Infinity symbol should have a descriptive alt text.
  [ ] Program Area images should be `alt=""` and is related to issue [Review the alt text for program-area page images and ensure that they adhere to WCAG #2354](https://github.com/hackforla/website/issues/2354). The alt text also impacts the images on the Program Areas page.
[ ] Complete writing tech dropdown issue: https://github.com/hackforla/website/issues/2972
[ ] Finish issue [Adding alt="" to guide card images on Toolkit page to follow WCAG](https://github.com/hackforla/website/issues/2929)
[ ] Update wiki dr adopted
    * [ ] Template
    * [ ] Instructions
[ ] 🚩 Update dr adopted and not implemented
  [ ] Update wiki dr not implemented
      * [ ] Instructions
  [ ] Update wiki dr adopted
      * [ ] Template
      * [ ] Instructions
 [ ] 🚩 [Sun All Leads Tasks Mar 6](https://github.com/hackforla/website/issues/2699#issuecomment-1060041771)
  [ ] Figure out how to incorporate 3DPO data into our repo. See this for all the links: https://hackforla.slack.com/archives/D02RPFKGSPQ/p1646603992466019
  [X] 🚩 From issue [homepage launch #2481](github.com/hackforla/website/issues/2481)
[ ] 🚩 [Sun Dev Apr 10](https://github.com/hackforla/website/issues/2027#issuecomment-1090787884)
  [ ] Check Join Us alt issue
  [ ] Redo Donate alt issue and double check
  [X] Follow up with this: https://github.com/hackforla/website/issues/2392#issuecomment-1094342204
  [X] Update and check this vague issue https://github.com/hackforla/website/issues/2173.
  [X] Review spreadsheet for Devin's issue https://github.com/hackforla/website/issues/2061 requires clarification
